### DISCUSSION TITLE
Webhook error [Jun 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2322] [Discord Threads]
> 🙋‍♂️ **Bergi9**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@269909921724629004> Error appears to be internal to NocoDB. You can ignore. Let us know NocoDB version & way to reproduce if you can to help us fix it.  
#  
> 🙋‍♂️ **Bergi9**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **pg**   
#### ProjectOnRootDB: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.91.1**   
####    
#### Webhook url trigger with `{{ json data }}` as body.  
#  
> 👨‍🦲 **dstala**  
#### <@269909921724629004> need a favour. try & let us know if you still find this issue on latest release (v0.91.7).  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981846066406244372)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
